sdk/keyvault/Azure.Security.KeyVault.Administration/src/RestoreOperation.cs

```diff
@@ -56,6 +56,22 @@ internal RestoreOperation(KeyVaultBackupClient client, ResponseWithHeaders<Servi
             _retryAfterSeconds = response.Headers.RetryAfter;
         }
 
+        /// <summary>
+        /// Initializes a new instance of a RestoreOperation.
+        /// </summary>
+        /// <param name="client">An instance of <see cref="KeyVaultBackupClient" />.</param>
+        /// <param name="response">The <see cref="ResponseWithHeaders{T, THeaders}" /> returned from <see cref="KeyVaultBackupClient.StartSelectiveRestore(string, Uri, string, string, CancellationToken)"/> or <see cref="KeyVaultBackupClient.StartSelectiveRestoreAsync(string, Uri, string, string, CancellationToken)"/>.</param>
+        internal RestoreOperation(KeyVaultBackupClient client, ResponseWithHeaders<ServiceSelectiveKeyRestoreOperationHeaders> response)
+        {
+            Argument.AssertNotNull(client, nameof(client));
```

Comment:
Nit: keeping in mind this is internal - but you documented it anyway - you *might* want to document the exceptions as well.